client/lib/analytics/utils/may-we-track-current-user-gdpr.ts

```diff
@@ -0,0 +1,26 @@
+import cookie from 'cookie';
+import debug from './debug';
+import getCountryCodeFromCookies from './get-country-code-from-cookies';
+import isCountryInGdprZone from './is-country-in-gdpr-zone';
+
+/**
+ * Returns a boolean telling whether we may track the current user.
+ *
+ * WARNING: this function only works on the client side.
+ *
+ * @returns Whether we may track the current user
+ */
+export default function mayWeTrackCurrentUserGdpr(): boolean {
```

Comment:
Note: I tried to preserve the logic in this method, because it's not used by the GDPR banner, but is used in other parts of the application. It should behave the same as before, but please double-check my work to make sure I didn't miss something.